.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseMergedDateTimeParser.cs

```diff
@@ -323,6 +323,26 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject refTime)
                 }
             }
 
+            /* modification of "past datetime" references
+               Input : 22 april at 5 pm. (reference time is 22/04/2022 T17:30:00)

```

Comment:
It would be better to have a textual explanation of the behaviour. But I won't block the merge due to this.
Please add the explanation in a future PR.

Reply:
The example can stay in the file with the modification method code.

Reply:
resolve in PR #3007